esx_service/vmdk_ops.py

```diff
@@ -640,6 +643,21 @@ def dev_info(unit_number, pci_slot_number):
     return {'Unit': str(unit_number),
             'ControllerPciSlotNumber': pci_slot_number}
 
+def reset_vol_meta(vmdk_path):
+    '''Clears metadata for vmdk_path'''
+    vol_meta = kv.getAll(vmdk_path)
+    if not vol_meta:
+        vol_meta = {}
+    logging.debug("Reseting meta-data for disk=%s", vmdk_path)
+    if vol_meta.keys() & {kv.STATUS, kv.ATTACHED_VM_UUID, kv.ATTACHED_VM_NAME}:
```

Comment:
I tried this line after setting vol_meta to {}, and I am getting the following exception: 
`TypeError: unsupported operand type(s) for &: 'list' and 'set'`